sql/migrate/migrate.go

```diff
@@ -182,10 +184,11 @@ type (
 		PartialHashes []string
 		// OperatorVersion holds a string representation of the Atlas operator managing this database migration.
 		OperatorVersion string
-		// Meta holds additional custom meta-data given for this migration.
-		Meta map[string]string
 	}
 
+	// RevisionType defines the type of the revision record in the history table.
+	RevisionType uint
```

Comment:
Why not use booleans ? We lose the capability to use the db for filtering nicely this way . And disk  isn't a major concern here

Reply:
Because the state is not binary here. There are baseline, exec, resolved, and execs that were resolved. 

Also:
<img width="523" alt="image" src="https://user-images.githubusercontent.com/7413593/182906793-9a8b7e27-31b2-4a41-93d7-63c22a2f33b4.png">


We can use `GoType` to avoid this convertion, but I didn't feel it's needed atm - we can always change this, it's an application thing.